### ISSUE TITLE
IoCreateSymbolicLink fails when creating a symbolic link to a Drive Letter.

### ISSUE THREAD

**Issue by LukeUsher** [Cxbx-Reloaded/Cxbx-Reloaded#490]
This can be observed with XOnlineDash (Version 4920, XDK 5233), the following call triggers a "Could not map D:" error.
``` 
IoCreateSymbolicLink(
   SymbolicLinkName     : 0x0BCAFC0C -> STRING* {
   .Length               : 6
   .MaximumLength        : 7
   .Buffer               : (char *)0x0BCAFBF8 = "\\??\\b:"}
   DeviceName           : 0x0BCAFC14 -> STRING* {
   .Length               : 2
   .MaximumLength        : 3
   .Buffer               : (char *)0x0BCAFC38 = "D:"}
);
```

**Comment by LukeUsher** [comment]
As this is a Kernel bug that effects multiple titles, and will continue to be a problem even with LLE, I think it's fair to mark this as High Priority.

**Comment by LukeUsher** [comment]
This has been solved as part of my xonline-tweaks branch, and will be merged when I finish my work there.

**Comment by LukeUsher** [comment]
Solved as of #492 

Note that there are still some titles that trigger 'Could Not Map' but that is a different issue, see #45
